examples/getting-started/serverless.yml

```diff
@@ -44,9 +42,6 @@ provider:
         - Effect: Allow
           Resource: !GetAtt Queue.Arn
           Action: [sqs:GetQueueUrl, sqs:GetQueueAttributes, sqs:SendMessage, sqs:receiveMessage, sqs:DeleteMessage, sqs:PurgeQueue]
-        - Effect: Allow
-          Resource: arn:aws:ssm:${aws:region}:${aws:accountId}:parameter/${self:service}-${sls:stage}/*
```

Comment:
Oh wait you raise a good point, I'll double check that! Thanks!

Reply:
So I think we are good for most of the users:

- those using `${ssm:...}` will not need extra permissions
- those using `bref-ssm:/...` will need to add permissions in `serverless.yml`

(docs: https://github.com/brefphp/bref/blob/v2/docs/environment/variables.md#retrieving-secrets)

The users using the second option will be more advanced users (it's documented to be an advanced option), and they will have the docs anyway (once v2 is published, see link above), so I wouldn't add that to the example as it might be more confusing than helping.